paasta_tools/contrib/create_paasta_playground.py

```diff
@@ -0,0 +1,57 @@
+import json
+import os
+
+from render_template import render_values
+
+
+def main():
+    cluster = os.environ.get(
+        "PAASTA_TEST_CLUSTER", "kind-{USER}-k8s-test".format(**os.environ)
+    )
+    config_path = "etc_paasta_playground"
+
+    # create an etc_paasta_playground directory if it doesn't exist
+    # and copy fake_etc_paasta content into etc_paasta_playground directory
+    if not os.path.isdir("etc_paasta_playground"):
+        os.mkdir("etc_paasta_playground")
+
+    render_values(
+        "./k8s_itests/deployments/paasta/fake_etc_paasta", "etc_paasta_playground", None
+    )
+
+    # Generate Cluster name
+    clusters_names = {"clusters": [cluster], "cluster": cluster}
+    clusters_path = os.path.join(config_path, "clusters.json")
+    with open(clusters_path, "w") as f:
+        json.dump(clusters_names, f)
+
+    # Add in volumes.json
+    hacheck_sidecar_volumes = {"volumes": [], "hacheck_sidecar_volumes": []}
+    volumes_path = os.path.join(config_path, "volumes.json")
+    with open(volumes_path, "w") as f:
+        json.dump(hacheck_sidecar_volumes, f)
+
+    # create a fake_soa_config directory if it doesn't exist
+    # and copy fake_soa_config content into soa_config_playground directory
+    if not os.path.isdir("soa_config_playground"):
+        os.mkdir("soa_config_playground")
+    render_values(
+        "./k8s_itests/deployments/paasta/fake_soa_config", "soa_config_playground", None
```

Comment:
suggestion: we could use kwargs here to make this a little more self-documenting: 
```suggestion
        src="./k8s_itests/deployments/paasta/fake_soa_config", dst="soa_config_playground", values=None
```
since this function's name doesn't really give the reader a hint as to what these strings are for

Reply:
that said, it would be nice to switch to an actual templating library like jinja2 or mako in the future rather than carrying this function forward :p